### PR TITLE
breaks long long lines for code and dt blocks

### LINKED ISSUE #270: Sizing problem in Firefox
I usually use Chrome, but opened FF the other day and found something off with the sizing:

This is full-screen in Firefox (`56.0.1 (64-bit)`). Note the X-scroll:
firefox_fullscreen
As the spacing indicates, even when scrolled all the way right, there's not room for the side spacer.

I had a fiddle with the media queries, thinking it might be something to do with [this](https://github.com/amiaopensource/ffmprovisr/blob/fe937d7e4a8f9482ee2a5e44b6392c722183a43e/css/css.css#L60).  
Toggling off `max-width: 70%;` improved things a little, but didn't fix the problem. It looks like this is happening at the `body` level, anyway.

With `max-width: 70%;` turned off:
firefox_fullscreen_mq

For comparison, here's how it looks in Chrome (`62.0.3202.62`):
chrome_fullscreen

@ablwr, can you offer your CSS expertise?

### PR DESCRIPTION
**Pull request by ablwr**
Unlike #274, resolves #270 by breaking up the very stubborn code blocks which hurt Firefox!

### CODE REVIEW

File: css/css.css [hunk 1]
```diff
@@ -120,6 +120,8 @@ code {
   border-radius: 4px;
   word-break: break-all;
   word-wrap: break-word;
+  white-space: normal;
```
Comment (retokromer) [inline css/css.css:123]: Is `normal` not the default value of `white-space`?

**Comment by ablwr** [comment]
I think the `<code>` blocks are behaving differently (on Firefox only) so we must be explicit here.

**Comment by retokromer** [comment]
I’ve just checked the specs. It’s indeed the default. Firefox is buggy.

**Review by retokromer — APPROVED** [review]

**Comment by ablwr** [comment]
I don't disagree!

**Pull request merged**